Automattic/jetpack
# My Jetpack: tweak height of Jetpack Logo

My Jetpack: tweak height of Jetpack Logo

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: tweak height of Jetpack Logo

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to My Jetpack overview
* Confirm logo hright is 40px

before | after
-----|------
<img width="399" alt="image" src="https://user-images.githubusercontent.com/77539/155538670-86fa3d9b-f4ce-4197-9352-db2a514761dc.png"> | <img width="396" alt="image" src="https://user-images.githubusercontent.com/77539/155538201-53277a01-62e1-4dce-ac5e-a3b8d90188f1.png">
